### PR TITLE
Add a simple document about the format of the BM file

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

**Review by rebtoor — APPROVED** [review]

File: docs/source/baremetal/01_baremetal_hosts_data.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Baremetal hosts information
+
+In order to make the framework aware of the existence of external
+baremetal nodes that should be taken into consideration the user should
+provide details about those nodes.
+
+~~~{tip}
+For fully virtualized environments the framework takes care of automatically
+generating these details for the user.
```
Comment (cjeanner) [inline docs/source/baremetal/01_baremetal_hosts_data.md:9]:
```suggestion
For fully virtualized environments the framework automatically
generates these details for the user.
```

**Comment by cjeanner** [comment]
/approve

thanks for that doc update! Be sure to advertise it downstream for QE and other consumers once the feature is fully in!

**Comment by pablintino** [comment]
> /approve
> 
> thanks for that doc update! Be sure to advertise it downstream for QE and other consumers once the feature is fully in!

Luckily for them them we will directly showcase this the NFV VA, so we will provide them a real example :)

**Review by psathyan — APPROVED** [review]

**Review by lewisdenny — APPROVED** [review]
/lgtm

**Pull request merged**